cogs/owner.py

```diff
@@ -57,33 +56,30 @@ async def unload(self, ctx, cog: str):
         try:
             await self.bot.unload_extension(f"cogs.{cog}")
         except Exception as e:
-            embed=discord.Embed(title="Unload", description=f"Could not unload the `{cog}` cog.")
-            await ctx.send(embed=embed)
-            return
+            embed=discord.Embed(title="Error!", description=f"Could not unload the `{cog}` cog.")
+            await context.send(embed=embed) return
         embed=discord.Embed(title="Unload", description=f"Successfully loaded the `{cog}` cog.")
-        await ctx.send(embed=embed)
+        await context.send(embed=embed)
 
     @commands.hybrid_command(
         name="reload",
         description="Reloads a cog.",
     )
     @checks.is_owner()
-    async def reload(self, ctx, cog: str):
+    async def reload(self, context: Context, cog: str):
```

Comment:
Please add the
```py
@app_commands.describe()
```
decorator for the `cog` parameter.

Reply:
what does this do?

Reply:
Gives a description of the argument you need to provide when using the command, ideally the same as in the command description, so same as
```py
:param ...
```